ospo-book/content/en/04-chapter.md

```diff
@@ -1,90 +1,128 @@
+| Define open source compliance rules and practices                                                                          | An explicit consensus on the organization's open source compliance rules and practices between the legal and business stakeholders.            | The organization knows that it has a managed approach to the legal aspects of open source consumption, which can be maintained and improved over time. Each company has different aspects of open source compliance, interpretations of licenses and different risk appetite (e.g dealing with regulations). Having well-defined compliance rules and practices is the first step toward deterministic open source compliance                                                                                                                                               |
+| Define rules and policies on using open source (criteria for using open source software which relate to open source health)                     | Consumption of open source projects is not just viewed through the compliance lens, but is considered more holistically and includes the risks associated with unhealthy projects. A consensus is built in the company related to the hygiene related to consumed open source components. The organization has clear policies to follow. | Consumed open source projects are lower in risk because they are healthy, fixing security vulnerabilities, implementing new features and release regularly.                                                                                                                                                                                                                                                                                         |
+| Define rules and policies on how to contribute to open source (criteria on how to engage in the community, how to transfer rights, CLAs) | The OSPO can increase awareness of the two-way relationship with open source projects. Using policies supports a consistent and ethical approach. The organization has clear policies to follow.                                                                                                     | Policies and practices ensure that the organization considers how to jointly build value with open source projects. Contributions made are likely to improve the company reputation, not damage it.                                                                                                                                                                                                                                                                                                                                 | 
```

Comment:
Vale nags about this if the abbreviation has not been written out fully in this file yet. We should do that.

Reply:
Done, thank you!